src/common/headlessService/utils.ts

```diff
@@ -237,6 +238,15 @@ export function isLanguage(
   return !!(<Language>language);
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isLinkItem(item: any): item is LinkItem {
```

Comment:
Maybe `item: unknown` instead of `item: any` might get rid of the `eslint-disable-next-line @typescript-eslint/no-explicit-any`?

Reply:
Boolean(item.url && item.title) actually filters away valid LinkItem values where url and title are empty strings i.e. Boolean("" && "") === false. Are you explicitly wanting only such LinkItem values that have non-empty url and titles? Although Boolean(" " && " ") === true. Typeguards are not necessarily exhaustive always so it's fine, just wanted to bring up that this filters away some LinkItem values.